### PR TITLE
create structure project

### PR DESCRIPTION
**Pull request by ACEof**
I created everything exept hot reload.

### CODE REVIEW

**Comment by abtv** [comment]
Please, write a simple test to check if tests setup is correct @ACEof

**Comment by ACEof** [comment]
Tests work correctly

File: example.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function square(a) {
+  return a * a;
+}
+
+module.exports = square;
```
Comment (abtv) [inline example.js]: @ACEof please use ES6 export

File: tests/example.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const square = require('../example');
```
Comment (abtv) [inline tests/example.test.js]: please use ES6 import

File: tests/example.test.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+test('square 5', () =>{
```
Comment (abtv) [inline tests/example.test.js]: please add meaningful description, like here: https://github.com/facebook/jest/blob/master/examples/getting-started/sum.test.js

File: tests/example.test.js (hunk 2 above)
Comment (ACEof) [reply tests/example.test.js]: Ok. I understood)

**Pull request merged**